setup/security/authentication/oauth/google/index.md

```diff
@@ -18,11 +18,41 @@ This page instructs you on how to obtain an OAuth 2.0 client ID and client secre
 ![GCP console to create OAuth 2.0 client screenshot](gcp-oauth-client.png)
 
 
+## Configure Halyard
```

Comment:
I see this often, "configure Halyard," and I think: you're not configuring Halyard, you're _using_ Halyard to configure Spinnaker. Thoughts?

Reply:
I suppose, though to the extent that Halyard is an abstraction over configuring Spinnaker directly, I still think of it as configuring Halyard (then using Halyard to *deploy* Spinnaker).  But I'm also happy to use a different term here...I just can't really think of one that's better.

Reply:
Aight, stet. It's a small thing anyway. :)